packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -73,12 +73,10 @@ and want to associate all stacks in the `App` scope to `MyAssociatedApplication`
 ```ts
 const app = new App();
 const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
-    applicationName: 'MyAssociatedApplication',
-    description: 'Testing associated application',
-    stackProps: {
-        stackName: 'MyAssociatedApplicationStack',
-        env: {account: '123456789012', region: 'us-east-1'},
-    },
+  appAssociatorProps: [appreg.ApplicationAssociatorPropsInputFactory.getApplicationAssociatorPropsFromAppName('MyAssociatedApplication', {
```

Comment:
This line sets off a bit of red flags to me. Please talk in the customer's *domain language*, not in details of how the programming constructs work. Yes, I'm aware that in these case the customer is also a programmer, so there may be some who are not intimidated upon seeing the identifier `ApplicationAssociatorPropsInputFactory`. There will also be some that are, who may think "this is too complicated for me", and we don't want to lose those people.

I usually phrase this as "name a thing after what it **does**, not what it **is**". So yes, while this class may **be** a "factory", the user doesn't give two licks whether it is a factory or not. They also wouldn't care if it was a "singleton", or if it was a "visitor". They only thing they care about is: "what happens when I *use* this class".

```
  appAssociatorProps: 
               ^^^^^ Why is this called "props" here? 
      What value does it add to a user to say these are props?
      We are already IN a props object to an appAssociator. Why do
      we have "appAssociatorProps" at a deeper level?

      I think this is looking for a different term.

    [appreg.ApplicationAssociatorPropsInputFactory.
                              ^^^^^^^^^^^^^^^^^^^
                           Same here. Techincally speaking btw the
                           static method is the "factory", not the class
                           that holds it.
                                 
    getApplicationAssociatorPropsFromAppName(
    ^^^^^ Don't start this with get. First of all, jsii won't let you,
        and get is generally just a bad word to use for anything that's
        not a property access in Java.
```

We can do shorter and sweeter:

```ts
const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
  applications: [
    appreg.Application.existingApplicationByArn('arn:aws:....:MyAssociatedApplication'),
    appreg.Application.existingApplicationByName('myname', {
      account: '1234', // Optional, defaults to current? Not sure if that's a good idea
      region: 'us-east-1',
    }),
    appreg.Application.createApplicationStack({
       stackName: 'MyAssociatedApplicationStack', // Optional
       account: '123456789012',  // Optional, defaults to whatever? 
       region: 'us-east-1'
    }),
  ]
});
```

Now, obviously `appreg.Application` already exists, so we need to find an alternative name for that. But keep it short and sweet and to the point and meaningful to the user please.